src/frontend/components/Navbar/Navbar.js

```diff
@@ -1,51 +1,69 @@
 import React, { useState } from "react";
 import { NavLink } from "react-router-dom";
-
+import { useAuth } from "../../context";
 import "./Navbar.css"
 
+
+
 function Navbar() {
-  const [accountList, setAccountList] = useState(false);
+    const [listVisibility, setListVisibility] = useState(false);
+    const { auth, setAuth } = useAuth();
+
+    const signOutHandler = (setAuth) => {
+        localStorage.removeItem("AUTH_TOKEN");
+        localStorage.removeItem("USERNAME");
+        setAuth((auth) => ({
+            ...auth,
+            status: false,
```

Comment:
instead of 'status' you can name this variable as 'isAuth'.

Reply:
done